src/lib/Libwin/accesinfo.c

```diff
@@ -1197,8 +1044,7 @@ make_dir_files_everyone_read(char *path)
 		make_dir_files_everyone_read(dirfile);
 	}
 	if (errno != 0 && errno != ENOENT) {
-		sprintf(logb,"readdir error; %s", path);
-		log_err(-1, "make_dir_files_everyone_read", logb);
+		log_eventf(PBSEVENT_ERROR, 0, LOG_ERR, __func__, "readdir error; %s", path);
```

Comment:
Not a comment, But just want to know where errno is getting set, and what may set it to ENOENT

Reply:
Except malloc didn't find the place where we are setting the errno

Reply:
Getting set in lstat